test/integration/experiments_from_simulation_scenario_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'minitest/autorun'
+require 'test_helper'
+require 'mocha/test_unit'
+
+require 'db_helper'
+require 'controller_integration_test_helper'
+
+class ExperimentsFromSimulationScenarioTest < ActionDispatch::IntegrationTest
+  include DBHelper
+  include ControllerIntegrationTestHelper
+
+  def setup
+    super
+    authenticate_session!
+  end
+
+  def teardown
+    super
+  end
+
+  test 'get experiment ids from one simulation scenario' do
+
+    simulation = Simulation.new({
+                                    'name' => 'test',
+                                    'user_id' => @user.id,
+                                    'created_at' => Time.now
+                                })
+    simulation.save
+
+    exp = ExperimentFactory.create_experiment(@user.id, simulation, name: 'exp')
```

Comment:
Czy mógłbyś dodać jeszcze do tego testu wrzucenie do bazy eksperymentu, który ma inną symulację? Z tego co widzę, w tym momencie ten test sprawdza, czy pobierane są wszystkie eksperymenty z bazy, a potrzebujemy udowodnić, że to są tylko przypisane dla danej symulacji.